Main/StackWalker/StackWalker.cpp

```diff
@@ -81,124 +81,115 @@
  *   SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  *
  **********************************************************************/
-#include <windows.h>
-#include <tchar.h>
 #include <stdio.h>
 #include <stdlib.h>
-#pragma comment(lib, "version.lib")  // for "VerQueryValue"
-#pragma warning(disable:4826)
+#include <tchar.h>
+#include <windows.h>
+#pragma comment(lib, "version.lib") // for "VerQueryValue"
+#pragma warning(disable : 4826)
 

```

Comment:
@JochenKalmbach, it was performed by clang-format.
Since I specified `BasedOnStyle: LLVM` in `.clang-format`, it should order the headers following [LLVM coding standards - #include style](http://llvm.org/docs/CodingStandards.html#include-style)
The result is a bit puzzling because headers should be ordered according to the order below but it's not the case.
1. Main Module Header
2. Local/Private Headers
3. LLVM project/subproject headers (clang/..., lldb/..., llvm/..., etc)
4. System #includes